modules/azuredevops/project/backplane/main.tf

```diff
@@ -0,0 +1,73 @@
+data "azurerm_client_config" "current" {}
+
+data "azurerm_subscription" "current" {}
+
+# Create a service principal for Azure DevOps management
+resource "azuread_application" "azure_devops" {
+  display_name = var.service_principal_name
+  description  = "Service principal for managing Azure DevOps projects and users"
+}
+
+resource "azuread_service_principal" "azure_devops" {
+  client_id = azuread_application.azure_devops.client_id
+}
+
+# Create a Key Vault to store the Azure DevOps PAT
+resource "azurerm_resource_group" "devops" {
+  name     = var.resource_group_name
+  location = var.location
+}
+
+resource "azurerm_key_vault" "devops" {
```

Comment:
not sure this should part of the backplane?

Reply:
because the vault is living then in the a managment sub? let us talk about that.